drupal/web/themes/custom/pori_events/src/scss/base/_typography.scss

```diff
@@ -0,0 +1,201 @@
+//
+// @file
+//
+// This file includes everything related to the basic typography
+//
+
+// ########## General ########## //
+body {
+  @include font-size-responsive($base-font-size-rem);
+  font-family: $text-font-family;
+  line-height: $base-line-height;
+}
+
+a {
+  color: $link-color;
+  text-decoration: none;
+  transition: all 150ms linear;
+
+  &:hover,
+  &:focus {
+    color: $link-color-hover;
+  }
+}
+
+strong {
+  font-weight: 700;
+}
+
+small {
+  font-size: 80%;
+}
+
+// ########## Headings ########## //
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: $heading-font-family;
+  margin-top: 0;
+  text-transform: uppercase;
+
+  a {
+    color: inherit;
+    font-weight: inherit;
+    text-decoration: none;
+  }
+}
+
+h1 {
+  font-size: $font-size-heading-1;
+  line-height: 1.1666;
+  margin-bottom: 3.15rem;
+
+  @media screen and (max-width: 599px) {
```

Comment:
Media-queries would be good to set up in a separate file and with variables.

Reply:
True, i took this straight from kada theme, seems to wrong in there. I'll change this.